server/datastore/mysql/hosts.go

```diff
@@ -1551,6 +1657,52 @@ ON DUPLICATE KEY UPDATE
 	return nil
 }
 
+func (ds *Datastore) generateAggregatedMDMSolutions(ctx context.Context, teamID *uint) error {
+	id := uint(0)
+
+	var results []fleet.AggregatedMDMSolutions
+	query := `SELECT
+				mdms.id,
+				mdms.server_url,
+				mdms.name,
+				COUNT(DISTINCT hm.host_id) as hosts_count
+			 FROM mobile_device_management_solutions mdms
+			 INNER JOIN host_mdm hm
+			 ON hm.mdm_id = mdms.id
+`
+	args := []interface{}{}
+	if teamID != nil {
+		args = append(args, *teamID)
+		query += ` JOIN hosts h ON (h.id = hm.host_id) WHERE h.team_id = ?`
+		id = *teamID
+	}
+	query += ` GROUP BY 1, 2, 3`
```

Comment:
```suggestion
	query += ` GROUP BY id, server_url, name`
```

? I didn't know about this shorthand, and when I googled it, all I can find is how bad it is :D